lib/zoho_hub/response.rb

```diff
@@ -47,6 +47,11 @@ def data
       data || @params
     end
 
+    def info
+      info = @params[:info] if @params.dig(:info)
+      info || @params
```

Comment:
I don't understand the purpose of this line :thinking:

Reply:
This method was created with reference to `def data` .

What about code like the following?

```
# return nil or info object
def info
  @params[:info]
end
```